### PR TITLE
Calculator component

### PR DESCRIPTION
**Pull request by anagudelogu**
# In this Milestone I:

- [x] Created a React component Calculator.
- [x] Used class-based components.
- [x] Added a descriptive README.md.

Feel free to give any optional feedback and suggestions about the project and clean code in general, I'm always trying to improve and write professional code. 🤓

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
+import Calculator from './components/Calculator';
+
 function App() {
```
Comment (t-yanick) [inline src/App.js]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, " `Use class based components (All components should be class-based).` ", kindly change your `App.js` component which is currently a `functional component` to a `class-based` component.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @anagudelogu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (anagudelogu) [reply src/App.js]: @t-yanick Sure! thank you!

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @anagudelogu

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**